Model/AutoRoute.php

```diff
@@ -50,7 +50,7 @@ public function setType($type)
         $this->setDefault('type', $type);
     }
 
-    public function setRedirectTarget(AutoRouteInterface $redirectRoute)
+    public function setRedirectTarget($redirectRoute)
```

Comment:
No it's not related. It's only to have a green Travis build.
I don't know if you're aware of it but at this moment if you run tests from master branch of https://github.com/symfony-cmf/RoutingAutoBundle they don't pass because of the change in the AutoRouteInterface (https://github.com/symfony-cmf/RoutingAuto/commit/5d01aa16245b321adfabacfc00abd90021bc0bd8#diff-4f339ec2f4961f0ef07d166e318bf9bf).